.github/workflows/prbot.yml

```diff
@@ -49,10 +49,10 @@ jobs:
       matrix:
         spec:
           [
-            'cypress/integration/*',
-            'cypress/integration/result-list/**/*',
-            'cypress/integration/facets/**/*',
-            'cypress/integration/search-box/**/*',
+            'cypress/e2e/*',
+            'cypress/e2e/result-list/**/*',
+            'cypress/e2e/facets/**/*',
+            'cypress/e2e/search-box/**/*',
```

Comment:
It wasn't configurable to keep integration?

Reply:
That's what the migration tool recommend, since they now support 2 types of testing: end to end (like we are doing) and component testing (which unfortunately is mostly geared for React and Vue projects)

Reply:
I see!